src/target/EmailServiceTarget.php

```diff
@@ -93,7 +93,7 @@ public function export()
         );
 
         foreach ($this->messages as $msg) {
-            $body = $this->formatMessage($msg);
+            $body = sprintf("<pre>%s</pre>", $this->formatMessage($msg));
```

Comment:
You may actually want to wrap it just when passing it as the `html_body` on line 104 below. As-is, the `text_body` will also have that HTML.